apps/fabric-website/src/pages/Controls/TextPage/docs/android/TextImplementation.md

```diff
@@ -0,0 +1,27 @@
+### Style names
+
+`TextAppearance.UIFabric.Headline`
+
+`TextAppearance.UIFabric.Title1`
+
+`TextAppearance.UIFabric.Title2`
+
+`TextAppearance.UIFabric.Heading`
+
+`TextAppearance.UIFabric.SubHeading`
+
+`TextAppearance.UIFabric.SubHeading2`
+
+`TextAppearance.UIFabric.Body1`
+
+`TextAppearance.UIFabric.Body2`
+
+`TextAppearance.UIFabric.Caption`
+
+### Library
+
+[UI Fabric Android](https://github.com/OfficeDev/ui-fabric-android)
```

Comment:
Should this be a more specific link?

Reply:
I think these are supposed to point to the library in general. I think the label supports that. If we wanted to point to source code, that'd be a separate label. @vladfilyakov do you see value in doing something like that or want to do something different with this link?

Reply:
I was thinking about this for some time. It probably does not make much sense to repeat a link to Fabric repo on each control page - we might have a more specific link to provide more value to a person reading these docs. Maybe link to a file that contains the main class (shown in "Control name"). It might be a problem when we have multiple controls on the same page (iOS cells, for example) though. In this case we might point the URL to a folder where those classes are, not specific file. But we might still need to have multiple links when controls are in different folders, like Activity Indicator and HUD on iOS. We do provide 2 links to 2 demos for them right now.

How complex will it be to have the following:

## Implementation
### Control name
`GenericCell`
`BooleanCell`
### Source code
[GenericCell](goes_to_source_code_file_for_GenericCell)
[BooleanCell](goes_to_source_code_file_for_BooleanCell)
### Sample code
[GenericCell sample](goes_to_demo_code_file_for_GenericCell)
[BooleanCell sample](goes_to_demo_code_file_for_BooleanCell)

Might be a different PR altogether. Thoughts? "Control name" section might be not needed anymore in this case.

Reply:
Yeah I think those extra links add good value and agree that we probably don't need "Control name" at that rate. I like the idea of doing it in a separate PR, too. I'll make a work item to track it.